Fix docs to make property listing in the website docs more useful. (#5746)
chef/inspec: docs-chef-io/content/inspec/resources/opa_api.md

An OPA query as a JSON data file or a string in JSON format.

## Properties

### result

The `result` property checks whether the resource query returns an empty result.

    its('result') { should be nil }

### allow

> The `allow` property checks if specific input is as per the policy defined in OPA. If `allow` is not defined in the policy file then this matcher will not work.

Reviewer: ```suggestion
The `allow` property checks if a specific input matches the policy defined in OPA. This matcher will not work if `allow` is not defined in the policy file.
```
